### PR TITLE
endpoint to serve the application secret.

### PR DESCRIPTION
**Pull request by ttung**
This is the worst-kept secret!

### CODE REVIEW

File: tests/test_application_secrets.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+pkg_root = os.path.abspath(os.path.join(os.path.dirname(__file__), '..'))  # noqa
+sys.path.insert(0, pkg_root)  # noqa
+
+import dss
+from tests.infra import DSSAsserts, start_verbose_logging
+
+start_verbose_logging()
+
+
+class TestFileApi(unittest.TestCase, DSSAsserts):
```
Comment (Mackey22) [inline tests/test_application_secrets.py]: Class name?

Comment (ttung) [inline tests/test_application_secrets.py]: LOL that's embarrassing.

**Comment by Mackey22** [comment]
Hmm build failed for some reason. Other than that lgtm

**Comment by kislyuk** [comment]
IMHO it's better not to declare this as part of the official API and instead shim it in the same way `/` is shimmed in here: https://github.com/HumanCellAtlas/data-store/blob/3bbfb8c79b7005b2b0f64d05b2c36482f0f450f2/chalice/app.py#L52-L56

**Pull request merged**